packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -298,7 +320,7 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
               />
             )}
             <EventListener type="keydown" listener={onKeyDown} target={document} />
-            <Provider theme={teamsTheme} target={document}>
+            <Provider theme={teamsTheme} target={document} tabIndex={0} style={{ outline: 'none' }}>
```

Comment:
why do you need to set the tabindex on the `Provider` ?

Reply:
Because `Provider` essentially represents body.